include/boost/geometry/strategies/geographic/area_geographic.hpp

```diff
@@ -0,0 +1,212 @@
+// Boost.Geometry (aka GGL, Generic Geometry Library)
+
+// Copyright (c) 2016 Oracle and/or its affiliates.
+// Contributed and/or modified by Vissarion Fisikopoulos, on behalf of Oracle
+
+// Use, modification and distribution is subject to the Boost Software License,
+// Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
+// http://www.boost.org/LICENSE_1_0.txt)
+
+#ifndef BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_AREA_GEOGRAPHIC_HPP
+#define BOOST_GEOMETRY_STRATEGIES_GEOGRAPHIC_AREA_GEOGRAPHIC_HPP
+
+#include <boost/geometry/formulas/area_formulas.hpp>
+#include <boost/geometry/formulas/thomas_inverse.hpp>
+//#include <boost/math/special_functions/atanh.hpp>
+
+namespace boost { namespace geometry
+{
+
+namespace strategy { namespace area
+{
+
+/*!
+\brief Geographic area calculation by trapezoidal rule plus integral
+approximation that gives the ellipsoidal correction
+
+}
+
+*/
+
+template
+<
+    typename PointOfSegment,
+    typename Strategy = void,
+    const std::size_t SeriesOrder = 2,
+    bool ExpandEpsN = true,
+    bool LongSegment = false,
+    typename Spheroid = void,
+    typename CalculationType = void
+>
```

Comment:
I'm taking back a remark about `Spheroid` for now. It's because in existing geographic strategies the `Spheroid` parameter exists. On the other hand there are no `Point` parameters, but that's because the existing strategies are distance and side strategies which might take two different `Point` types.

Therefore we could think about getting rid of `PointOfSegment`. Unfortunately `area()` algorithm have to store the state of area calculation so the existing area strategies define this type and `PointOfSegment` is needed for that. So if we wanted to get rid of `PointOfSegment` we'd have to modify the `area()` algorithm and existing area strategies. They'd have to define a template allowing the algorithm to figure out the strategy sumation state type from `Point`/`Geometry` (in C++11 it won't be needed) but this would be done on algorithm level, not user level. So this would be a breaking change. On the other hand area strategies would be consistent with other strategies not requireing `Point` template parameters.